src/coreclr/jit/ee_il_dll.cpp

```diff
@@ -1615,45 +1615,19 @@ const char16_t* Compiler::eeGetShortClassName(CORINFO_CLASS_HANDLE clsHnd)
     return param.classNameWidePtr;
 }
 
-const WCHAR* Compiler::eeGetCPString(size_t strHandle)
+void Compiler::eePrintFrozenObjectDescription(const char* prefix, size_t handle)
 {
-#ifdef HOST_UNIX
-    return nullptr;
-#else
-    char buff[512 + sizeof(CORINFO_String)];
-
-    // make this bulletproof, so it works even if we are wrong.
-    if (ReadProcessMemory(GetCurrentProcess(), (void*)strHandle, buff, 4, nullptr) == 0)
+    const int maxStrSize      = 128;
+    char16_t  str[maxStrSize] = {0};
```

Comment:
```suggestion
    char16_t  str[maxStrSize];
```
Nit, seems unnecessary. Feel free to ignore.